io/post_fv3.F90

```diff
@@ -199,6 +199,8 @@ subroutine post_run_fv3(wrt_int_state,grid_id,mype,mpicomp,lead_write, &
         if (grid_id > 1) then
           write(post_fname, '(A,I2.2)') trim(post_fname)//".nest", grid_id
         endif
+        post_fname = trim(fv3atm_output_dir)//trim(post_fname)
```

Comment:
@WenMeng-NOAA FYI. Since now fv3atm history files will be in fv3atm_output_dir, maybe UPP standalone can directly point to this output directory without any additional changes.

Reply:
@junwang-noaa Sounds good to me.